libkbfs/folder_branch_ops.go

```diff
@@ -1582,11 +1582,10 @@ func (fbo *folderBranchOps) SetInitialHeadFromServer(
 	}()
 
 	if md.IsReadable() && fbo.config.Mode() != InitMinimal {
-		// We will prefetch this as on-demand so that it triggers downstream
-		// prefetches.
-		fbo.config.BlockOps().Prefetcher().PrefetchBlock(
-			&DirBlock{}, md.data.Dir.BlockPointer, md,
-			defaultOnDemandRequestPriority)
+		// We `Get` the root block to ensure downstream prefetches occur.
```

Comment:
Previously we only fetched the root block itself right, without actually triggering "downstream" prefetches.  This is a conscious change to start prefetching the children of the root block as well?  (That's fine with me, just making sure I understand.)

Reply:
Correct. I think this new behavior is both more consistent and desirable.